### ISSUE TITLE
Makefile does not offer suggestions

### ISSUE THREAD

**Issue by msprotz** [CatalaLang/catala#96]
```
jonathan@absinthe:~/Code/catala/examples/us_tax_code (master) $ make section_121.run --debug
GNU Make 3.81
Copyright (C) 2006  Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.

This program built for i386-apple-darwin11.3.0
Reading makefiles...
Updating goal targets....
 File `section_121.run' does not exist.
Must remake target `section_121.run'.
dune exec --no-print-director ../../src/catala/catala.exe --  --language=en Makefile section_121.catala_en
dune exec --no-print-director ../../src/catala/catala.exe --  --language=en \
		Interpret \
		-s  \
		section_121.catala_en
catala: required argument FILE is missing
Usage: catala [OPTION]... BACKEND FILE
Try `catala --help' for more information.
make: *** [section_121.run] Error 1
```

not sure there's a good way of dealing with this, except perhaps

```
SCOPE?=PleasePickADefaultScopeViaTheSCOPEVariable
```

in the shared Makefile

**Comment by denismerigoux** [comment]
Good suggestion, will implement.

**Comment by denismerigoux** [comment]
Closed by 86e7c4f
